src/main/java/thePackmaster/cardmodifiers/frostpack/FrozenMod.java

```diff
@@ -0,0 +1,93 @@
+package thePackmaster.cardmodifiers.frostpack;
+
+import basemod.abstracts.AbstractCardModifier;
+import basemod.helpers.CardModifierManager;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.util.TexLoader;
+import thePackmaster.util.Wiz;
+
+public class FrozenMod extends AbstractCardModifier {
+    public static String ID = SpireAnniversary5Mod.makeID("FrozenMod");
+
+    private int originalCost;
+    private boolean originalIsCostModified;
+    private boolean hadRetain = false;
+    private static final Texture tex = TexLoader.getTexture(SpireAnniversary5Mod.modID + "Resources/images/ui/frozenOverlay.png");
+
+
+    @Override
+    public boolean canPlayCard(AbstractCard card) {
+        return false;
+    }
+
+    @Override
+    public void onInitialApplication(AbstractCard card) {
+
+        CardCrawlGame.sound.play("ORB_FROST_CHANNEL", 0.1F);
+        originalCost = card.cost;
+        originalIsCostModified = card.isCostModified;
+        if (card.costForTurn == 0) card.costForTurn++;
```

Comment:
This needs to be after the call to modifyCostForCombat instead of before -- right now it will make 0 cost cards into 2 cost cards.